migration/src/m20230725_220134_add_vdafs_and_query_types_to_aggregators.rs

```diff
@@ -0,0 +1,61 @@
+use sea_orm_migration::prelude::*;
+use serde_json::json;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, db: &SchemaManager) -> Result<(), DbErr> {
+        db.alter_table(
+            TableAlterStatement::new()
+                .table(Aggregator::Table)
+                .add_column(ColumnDef::new(Aggregator::Vdafs).json().null())
+                .add_column(ColumnDef::new(Aggregator::QueryTypes).json().null())
+                .to_owned(),
+        )
+        .await?;
+
+        db.exec_stmt(
+            Query::update()
+                .table(Aggregator::Table)
+                .values([
+                    (Aggregator::Vdafs, json!([1, 2, 3]).into()),
+                    (Aggregator::QueryTypes, json!([1, 2]).into()),
```

Comment:
Are there not constants somewhere for the VDAFs and query types? Or could we at least put comments explaining what `[1, 2, 3]` and `[1, 2`] mean?

Reply:
Migration doesn't depend on the app, but also there are not yet constants. I'll add a comment

Reply:
addressed in 428c6c52cd5647fa51d2dc1c57d381fe36449dd0